Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM p4lang/behavioral-model:latest
+ARG UBUNTU_VERSION=20.04
+ARG BASE_IMAGE=ubuntu:${UBUNTU_VERSION}
+FROM ${BASE_IMAGE}
```

Comment:
I think we need to keep `p4lang/behavioral-model:latest` as default base image.
This Dockerfile is used to automatically [build and publish](https://github.com/p4lang/p4c/blob/3ae0821aaff1b8eafb4caf4485cc602de2910c26/.github/workflows/ci-container-image.yml) the p4c container image in [Docker Hub](https://hub.docker.com/r/p4lang/p4c). This container image is being used in other projects.

Reply:
We could keep this docker file for the release build. With David's new setup we might not need a docker file at all for the p4c image.